runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7519,7 +7519,9 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
          }
       }
 #if defined(J9VM_OPT_JITSERVER)
-   bool cannotDoRemoteCompilation = cannotPerformRemoteComp(
+   TR_ResolvedMethod *resolvedMethod = fe->createResolvedMethod(&trMemory, (TR_OpaqueMethodBlock *)method);
+   bool forcedLocal = _compInfo.methodToBeLocalCompiled(resolvedMethod->signature(&trMemory),resolvedMethod->convertToMethod()->methodType());
```

Comment:
Based on this code, I think that `forcedLocal` gets the opposite value of what you want.
I also suggest testing `TR::Options::getDebug()` so that we don't create a resolved method for nothing (in most cases Debug will be null)

Reply:
Thanks. Renamed the variable and used the `debug` object.